app/javascript/components/miq-alert-set-form/helper.js

```diff
@@ -20,7 +20,7 @@ const findLabel = (emsId, mode) => {
 
 const changeTitle = (emsId, mode) => {
   const titleName = findLabel(mode, emsId);
-  const title = _(`Available ${titleName} Alerts`);
+  const title = sprintf(__('Available %s Alerts:'), titleName);
```

Comment:
This one is going to confusing for translators - Even I'm not sure what this is.  Can you give examples?  Wondering if we can change the string to make it clearer.

Reply:
Here if you change the 'Type` drop down above these fields you'll see the different titleName options. If i recall correctly those are already translated but i can double check

Reply:
I'm tempted to not show the title at all if you choose a type I'm not sure we care if it says "Available <type> Alerts" as opposed to just "Available Alerts"